server/stream.go

```diff
@@ -2389,7 +2395,7 @@ func (mset *stream) scheduleSetupMirrorConsumerRetryAsap() {
 	}
 	// To make *sure* that the next request will not fail, add a bit of buffer
 	// and some randomness.
-	next += time.Duration(rand.Intn(50)) + 10*time.Millisecond
+	next += time.Duration(rand.Intn(10000000)) + 10*time.Millisecond
```

Comment:
Let's make it clear here.

`rand.Intn(10*time.Millisecond)` with appropriate casts here.

Reply:
Done